framework/src/Volo.Abp.Http.Client.Dapr/Volo/Abp/Http/Client/Dapr/AbpHttpClientDaprModule.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Volo.Abp.Dapr;
+using Volo.Abp.Modularity;
+
+namespace Volo.Abp.Http.Client.Dapr;
+
+[DependsOn(
+    typeof(AbpHttpClientModule),
+    typeof(AbpDaprModule)
+)]
+public class AbpHttpClientDaprModule : AbpModule
+{ 
+    public override void ConfigureServices(ServiceConfigurationContext context)
+    {
+        var daprClientOptions = context.Services.ExecutePreConfiguredActions<AbpDaprClientOptions>();
+
+        PreConfigure<AbpHttpClientBuilderOptions>(options =>
```

Comment:
Shouldn't we write `PreConfigure` into `PreConfigureServices` method of the module class?

Reply:
we usually call the `AddHttpClientProxies` in the `ConfigureServices` method of `HttpClientModule`, so it can be ensure that `PreConfigure<AbpHttpClientBuilderOptions>` is executed before configuring http leint proxy 

Write into `ConfigureServices` can allow developer to pre configure the `AbpDaprClientOptions`